scripts/upgrade.sh

```diff
@@ -81,8 +81,8 @@ replace_binary() {
 
   set +e
 
-  NEW_BIN_SEMVER="$($NEW_BINARY -v | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+')"
-  FULL_BIN_SEMVER="$($FULL_BIN_PATH -v | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+')"
+  NEW_BIN_SEMVER="$($NEW_BINARY -v | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+' | sed 's/^v//')"
+  FULL_BIN_SEMVER="$($FULL_BIN_PATH -v | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+' | sed 's/^v//')"
```

Comment:
Just get the first line (the k3s version) and ignore the second line (with golang version)
```suggestion
  NEW_BIN_SEMVER="$($NEW_BINARY -v | head -1 |grep -Eo '[0-9]+\.[0-9]+\.[0-9]+')"
  FULL_BIN_SEMVER="$($FULL_BIN_PATH -v | head -1 | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+')"
```